requirements/requirements_git.txt

```diff
@@ -1,5 +1,5 @@
 git+https://github.com/ansible/system-certifi.git@devel#egg=certifi
 git+https://github.com/ansible/ansible-runner.git@devel#egg=ansible-runner
 awx-plugins-core @ git+https://github.com/ansible/awx-plugins.git@devel#egg=awx-plugins-core[credentials-github-app]
-django-ansible-base @ git+https://github.com/ansible/django-ansible-base@devel#egg=django-ansible-base[rest-filters,jwt_consumer,resource-registry,rbac,feature-flags]
+django-ansible-base @ git+https://github.com/zkayyali812/django-ansible-base@AAP-49910#egg=django-ansible-base[rest-filters,jwt_consumer,resource-registry,rbac,feature-flags]
```

Comment:
Shouldn't the checks still pass, and everything still be functional, if you left this at the current devel branch? If not then maybe there's something I'm not following.